Palo Alto Networks/paloalto-ngfw/_meta/smart-descriptions.json

```
@@ -717,5 +717,22 @@
         "type": "queried dns resolution for"
       }
     ]
+  },
+  {
+    "value": "{user.name} perform {action.name} with {action.outcome} on {event.dataset}",
+    "conditions": [
+      {
+        "field": "user.name"
+      },
+      {
+        "field": "action.name"
+      },
+      {
+        "field": "action.outcome"
+      },
+      {
+        "field": "event.dataset"
+      }
+    ]
```

I suggest splitting the smart-description and be more precise:
```suggestion
    "value": "{user.name} {action.name} the configuration successfully",
    "conditions": [
      {
        "field": "user.name"
      },
      {
        "field": "action.name"
      },
      {
        "field": "event.outcome",
        "value": "success"
      },
      {
        "field": "event.dataset",
        "value": "config"
      }
    ],
    {"value": "{user.name} failed to {action.name} the configuration",
    "conditions": [
      {
        "field": "user.name"
      },
      {
        "field": "action.name"
      },
      {
        "field": "event.outcome",
        "value": "failure"
      },
      {
        "field": "event.dataset",
        "value": "config"
      }
    ]
```